src/main/java/frc/robot/constants/RobotConfig.java

```diff
@@ -1,23 +1,10 @@
 package frc.robot.constants;
-//variable things like default speeds
+/**
```

Comment:
Add a newline between the package/imports and the docstring for the class declaration.